app/routes/posts/form.jsx

```diff
@@ -0,0 +1,85 @@
+import { redirect } from "@remix-run/node"
+import {useState} from 'react'
+
+export const action = async ({request}) => {
+  const form = await request.formData();
+  const name = form.get('first-name')
+  const email = form.get('email')
+  const inPerson = form.get('in-person')
+  const online = form.get('online')
+  const pizza = form.get('pizza')
+
+  const fields = {name, email, inPerson, online, pizza}
+
+  const post = await db.post.create({data: fields})
+  
+  return redirect('/')
+}
+
+export default function Forms(){
+
+  const [isChecked, setIsChecked] = useState(false)
+
+
+function handleClick(e){
+  if(e.target.value === 'radio1'){
+    setIsChecked(true)
+  } else {
+    document.getElementById('pizza').checked = false
+    setIsChecked(false)
+  }
+}
+
+  return (
+    <>
+
+    <div id="form-header">
+    <h4>Join us downtown and online</h4>
+    </div>
+
+    <div id="form-content">
+    <form method='POST'>
```

Comment:
Checkout the `Form` component from `@remix/react`. The api is the same but there's some optimizations that we can take of later. Such as action handlers for transition states, ie submitting, success, error etc.